source/_integrations/generic_ip_camera.markdown

```diff
@@ -106,7 +106,7 @@ camera:
 
 ### Sharing a camera feed from one Home Assistant instance to another
 
-If you are running more than one Home Assistant instance (let's call them the 'host' and 'receiver' instances) you may wish to display the camera feed from the host instance on the receiver instance. You can use the [REST API](/developers/rest_api/#get-apicamera_proxycameraltentity_id) to access the camera feed on the host (IP address 127.0.0.5) and display it on the receiver instance by configuring the receiver with the following:
+If you are running more than one Home Assistant instance (let's call them the 'host' and 'receiver' instances) you may wish to display the camera feed from the host instance on the receiver instance. You can use the [REST API](/developers/rest_api/#get-apicamera_proxycameraentity_id) to access the camera feed on the host (IP address 127.0.0.5) and display it on the receiver instance by configuring the receiver with the following:
```

Comment:
This is a broken link as well...

Reply:
@frenck As I mentioned, this PR requires #13469 (which was just merged), so it should work now.

Reply:
To clarify, #13469 has `/developers/rest_api` redirect to `https://developers.home-assistant.io/docs/api/rest.html`, and the link https://developers.home-assistant.io/docs/api/rest.html#get-apicamera_proxycameraentity_id is correct.

Reply:
We should use a direct link when possible.